packages/neon/neon/lib/src/router.dart

```diff
@@ -29,19 +33,17 @@ class AppRouter extends GoRouter {
           navigatorKey: navigatorKey,
           initialLocation: const HomeRoute().location,
           redirect: (final context, final state) {
-            final account = accountsBloc.activeAccount.valueOrNull;
-
             final loginQrcode = LoginQrcode.tryParse(state.location);
-            if (loginQrcode != null) {
-              return LoginQrcodeIntermediateRoute(
-                serverURL: loginQrcode.server,
+            if (state.path == const LoginRoute().location && loginQrcode != null) {
```

Comment:
Why is this limited to when the user is already on the login page? They could have one account logged in and be on the home page and then just scan a login qr code.